client/geocode/index.js

```diff
@@ -50,41 +54,38 @@ function reverse(ll, callback) {
  * Suggestions!
  */
 
-function suggest(text, callback) {
-  var bingSuggestions, nominatimSuggestions, totalSuggestions;
-  log('--> getting suggestion for %s', text);
-  get('/geocode/suggest/' + text, function(err, res) {
-    if (err) {
-      log('<-- suggestion error %s', err);
-      callback(err, res);
-    } else {
-      log('<-- got %s suggestions', res.body.length);
-	bingSuggestions = res.body;
-//      callback(null, res.body);
-	get('http://nominatim.openstreetmap.org/search' +
-	    '?format=json&addressdetails=1&' +
-	    'viewbox=' + southWest[0] + ',' +
-	    northEast[1] + ',' + northEast[0] + ',' + southWest[1] +
-	    '&bounded=1' +
-	    'countrycodes=us&q=' + text, function (err, nRes) {
-		var inside = false;
-	    nominatimSuggestions = [];
-            for (var i = 0; i < nRes.body.length; i++) {
-//		inside = inside && (nRes.body[i].lng > southWest[0]);
-//		inside = inside && (nRes.body[i].lng < northEast[0]);
-//		inside = inside && (nRes.body[i].lat > southWest[1]);
-//		inside = inside && (nRes.body[i].lat < northEast[1]);
-//		if (inside) {
-                    nominatimSuggestions.push(nRes.body[i]);
-//		}
-            }
-	    callback(
-		null,
-		nominatimSuggestions.slice(0,2).
-		    concat(bingSuggestions.slice(0,3))
-	    );
-	});
-    }
-  });
+function suggestAmigo(text, callback) {
+
+    var list_address;
+    var parameter = {
+        'token': config.realtime_access_token() ,
+        'boundary.rect.min_lat': '36.155617833819',
```

Comment:
We should not hard code lat/lng values.
To get bounding box you should query this URL:
https://www.amigocloud.com/api/v1/users/1/projects/661/datasets/22492?token=<token>
and get from JSON: "boundingbox": "-123.607177734375 36.155617833819,-120.701293945312 38.826870521381",